website/content/v1.8/advanced/install-kubevirt.md

```diff
@@ -0,0 +1,401 @@
+---
+title: "Install KubeVirt on Talos"
+description: "This is a guide on how to get started with KubeVirt on Talos"
+---
+
+KubeVirt allows you to run virtual machines on Kubernetes. It runs with QEMU and KVM to provide a seamless virtual machine experience and can be mixed with containerized workloads. This guide explains on how to install KubeVirt on Talos.
+
+## Prerequisites
+
+For KubeVirt and Talos to work you have to enable certain configurations in the BIOS and configure Talos properly for it to work. 
+
+### Enable virtualization in your BIOS
+
+On many new PCs and servers virtualization is enabled by default. Please consult your manufacturer on how to enable this in the BIOS. You can also run KubeVirt from within a virtual machine. For that to work you have to enable Nested Virtualization. This can also be done in the BIOS.
+
+### Configure your network interface in bridge mode (optional)
+
+When you want to leverage [Multus](../kubernetes-guides/network/multus) to give your virtualm achines direct access to your nodes network, your bridge need to configured properly. This can be done by setting your network interface in bridge mode. You can look up the network interface name by using the following command:
```

Comment:
```suggestion
When you want to leverage [Multus](../kubernetes-guides/network/multus) to give your virtual machines direct access to your node network, your bridge needs to be configured properly. This can be done by setting your network interface in bridge mode. You can look up the network interface name by using the following command:
```